apps/spark/src/main/java/com/linkedin/openhouse/jobs/client/TablesClient.java

```diff
@@ -91,6 +92,11 @@ protected GetTableResponseBody getTable(TableMetadata tableMetadata) {
         null);
   }
 
+  public Map<String, String> getTableProperties(TableMetadata tableMetadata) {
```

Comment:
Not a blocker: But are we increasing number of getTable calls? Should we try optimize these calls ?

Reply:
+1, may not be trivial though. Can refactor to inject Table at TableOperation layer itself in a separate PR.

Reply:
yes for now, this will become cheap after merged with DLO PRs